.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "cSpell.words": [
+    "tdata"
+  ]
+}
```

Comment:
[OPTIONAL] since this is your own VSCode setting. it doesn't necessarily need to be committed into the repo. so having this in the .gitignore file may be a good choice 👍

Reply:
@Bondok6 , thanks for the review I am going to fix that now